server/src/main/java/org/opensearch/action/bulk/TransportShardBulkAction.java

```diff
@@ -193,6 +196,14 @@ protected long primaryOperationSize(BulkShardRequest request) {
         return request.ramBytesUsed();
     }
 
+    @Override
+    protected ReplicationOverridePolicy overrideReplicationPolicy(IndexShard indexShard) {
```

Comment:
I believe `getReplicationOverridePolicy()` would be more appropriate (`overrideReplicationPolicy` assumes imperative action which should not return but apply)

Reply:
Makes sense.